docs/advanced/sbom/cyclonedx.md

```diff
@@ -167,9 +333,10 @@ $ cat result.json | jq .
 }
 
 ```
+
 </details>
 
 !!! caution
-    It doesn't support vulnerabilities yet, but installed packages.
```

Comment:
This tab is required.
https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage